### ISSUE TITLE
integer overflow in L1TGMT

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#27554]
When running workflow 134.804 step 3 in the UBSAN IB there is the error

```
/build/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/a33d565f9262c93f1d0ea38c818d514b/opt/cmssw/slc7_amd64_gcc700/cms/cmssw/CMSSW_11_0_UBSAN_X_2019-07-12-2300/src/DQM/L1TMonitor/src/L1TGMT.cc:205:40: runtime error: signed integer overflow: 152916275 * 3564 cannot be represented in type 'int'
```

which corresponds to
https://github.com/cms-sw/cmssw/blob/61ad0a34a8c61a073d9240fb9f111625ef561b8b/DQM/L1TMonitor/src/L1TGMT.cc#L205

**Comment by Dr15Jones** [comment]
assign l1

**Comment by perrotta** [comment]
This is fixed by #37534, as verified in CMSSW_12_4_UBSAN_X_2022-04-29-1100
